eng/Version.Details.xml

```diff
@@ -337,7 +337,7 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>4ac4c0367003fe3973a3648eb0715ddb0e3bbcea</Sha>
     </Dependency>
-    <Dependency Name="System.Security.Cryptography.Cng" Version="4.6.0" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
+    <Dependency Name="System.Security.Cryptography.Cng" Version="4.6.0" Pinned="true">
```

Comment:
@JunTaoLuo I believe this will need to be moved down with the other pinned dependencies. @mmitche / @wtgodbe please confirm that's still necessary.

Reply:
Also, double checking that CoherentParentDependency should be removed when we pin.

Reply:
Removing is fine.

Reply:
> @JunTaoLuo I believe this will need to be moved down with the other pinned dependencies. @mmitche / @wtgodbe please confirm that's still necessary.

Not after the CPD fix.  It can stay where it is.